.github/CODEOWNERS

```diff
@@ -472,34 +485,41 @@
 /core/templates/app.constants.ajs.ts @srijanreddy98
 /core/templates/combined-tests.spec.ts @srijanreddy98
 /core/templates/pages/interaction-specs.constants.ajs.ts @jameesjohn @vojtechjelinek
-/core/templates/pages/interaction-specs.constants.ts @nithusha21
+# TODO(#12694): Add @nithusha21 as owner after 2021-05-08
+/core/templates/pages/interaction-specs.constants.ts @kevintab95
 # TODO(#12656): Revert ownership to @brianrodri after 2021-05-18.
 /core/templates/pages/login-page/ @vojtechjelinek
 # TODO(#12656): Revert ownership to @brianrodri after 2021-05-18.
 /core/templates/pages/logout-page/ @vojtechjelinek
-/core/templates/I18nFooter.ts @nithusha21
-/core/templates/i18n-footer.directive.html @nithusha21
+# TODO(#12694): Add @nithusha21 as owner after 2021-05-08
+/core/templates/I18nFooter.ts @kevintab95
+# TODO(#12694): Add @nithusha21 as owner after 2021-05-08
+/core/templates/i18n-footer.directive.html @kevintab95
 /core/templates/app-events/ @srijanreddy98
 /core/templates/services/app.service*.ts @srijanreddy98
 /core/templates/services/contextual/document-attribute-customization.service*.ts @srijanreddy98
 /core/templates/services/contextual/meta-tag-customization.service*.ts @srijanreddy98
 /core/templates/services/interaction-rules-registry.service*.ts @DubeySandeep
 /core/templates/services/interaction-specs.service*.ts @DubeySandeep
-/core/templates/services/translation-file-hash-loader-backend-api.service.ts @nithusha21
+# TODO(#12694): Add @nithusha21 as owner after 2021-05-08
+/core/templates/services/translation-file-hash-loader-backend-api.service.ts @srijanreddy98
 /redis.conf @BenHenning
-/release_constants.json @DubeySandeep @nithusha21
+# TODO(#12694): Add @nithusha21 as owner after 2021-05-08
+/release_constants.json @DubeySandeep
 # TODO(#12656): Revert ownership to @brianrodri after 2021-05-18.
 /.firebase.json @vojtechjelinek
 
 
 # Miscellaneous.
-/__init__.py @nithusha21
-/core/__init__.py @nithusha21
-/core/controllers/__init__.py @nithusha21
-/core/domain/__init__.py @nithusha21
+# TODO(#12694): Add @nithusha21 as owner after 2021-05-08, lines 513-518
```

Comment:
You probably want "next 4 lines" or similar. The line numbers here aren't accurate and will change as people add todos. Ditto elsewhere.